app/posts/digital-public-works-ideas.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Ideas for digital public works
+description: Some thoughts on digital platforms and services that would be helpful for cross-government teams.
+date: 2024-03-01
+author:
+  name: Frankie Roberto
+  url: https://github.com/frankieroberto
+---
+
+A recent blog post from dxw suggests that [there needs to be more support for digital public works](https://www.dxw.com/2024/01/there-needs-to-be-more-support-for-digital-public-works/), described as platforms and tools which work across government to make it easier and faster to build services.
+
```

Comment:
Perhaps worth adding a definition of what a digital public work (or digital public good) is for those who aren’t aware. Could utilise the descriptions from the [Digital Public Goods Alliance](https://digitalpublicgoods.net/standard/).

Reply:
Agreed, and also maybe splitting into 2 shorter sentences

Reply:
Thanks!